### PR TITLE
SDCICD-915: support fips through rosaprovider

### PR DESCRIPTION
**Pull request by jbpratt**
fips is supported on ROSA and should be tested. This adds support for deploying fips enabled clusters through osde2e

Signed-off-by: Brady Pratt <bpratt@redhat.com>

### CODE REVIEW

File: pkg/common/providers/rosaprovider/cluster.go [hunk 1]
```diff
@@ -144,6 +144,10 @@ func (m *ROSAProvider) LaunchCluster(clusterName string) (string, error) {
 		}
 	}
 
+	if viper.GetBool(config.Tests.EnableFips) {
```
Comment (ryankwilliams) [inline pkg/common/providers/rosaprovider/cluster.go:147]: I know this config setting was already [present](https://github.com/openshift/osde2e/blob/main/pkg/common/config/config.go#L246) under test configuration. Do we think this should be a cluster specific setting under [cluster config](https://github.com/openshift/osde2e/blob/main/pkg/common/config/config.go#L265)?

Looks like this config setting is only used within the `pkg/e2e/verify/fips.go` test and there are 2 configs (fedramp/fips).

This change looks good to me :+1:, just wanted to mention this.

Comment (jbpratt) [reply pkg/common/providers/rosaprovider/cluster.go]: updated in b558e8a

Comment (ryankwilliams) [reply pkg/common/providers/rosaprovider/cluster.go]: Thanks for making the change @jbpratt!

**Pull request merged**